Projects/Presentation/Sources/CompanyDetail/CompanyDetailViewModel.swift

```diff
@@ -7,22 +7,55 @@ import Domain
 
 public final class CompanyDetailViewModel: BaseViewModel, Stepper {
     public let steps = PublishRelay<Step>()
+    public var companyID: Int?
     private let disposeBag = DisposeBag()
+    private let fetchCompanyInfoDetailUseCase: FetchCompanyInfoDetailUseCase
+    private let fetchReviewListUseCase: FetchReviewListUseCase
 
-    init( ) { }
+    init(
+        fetchCompanyInfoDetailUseCase: FetchCompanyInfoDetailUseCase,
+        fetchReviewListUseCase: FetchReviewListUseCase
+    ) {
+        self.fetchCompanyInfoDetailUseCase = fetchCompanyInfoDetailUseCase
+        self.fetchReviewListUseCase = fetchReviewListUseCase
+    }
 
     public struct Input {
+        let viewAppear: PublishRelay<Void>
         let recruitmentButtonDidTap: Signal<Void>
     }
 
-    public struct Output { }
+    public struct Output {
+        let companyDetailInfo: PublishRelay<CompanyInfoDetailEntity>
+        let reviewListInfo: PublishRelay<[ReviewEntity]>
+    }
 
     public func transform(_ input: Input) -> Output {
+        let companyDetailInfo = PublishRelay<CompanyInfoDetailEntity>()
+        let reviewListInfo = PublishRelay<[ReviewEntity]>()
+
+        input.viewAppear.asObservable()
```

Comment:
```suggestion
        input.viewAppear.asObservable()
                 .filter { $0 != nil }
```
이런식으로 널일때는 걸러주는게 맞는듯

Reply:
항상 true를 반환해준다는데 안 사용해도 괜찮은거 아니에요?

Reply:
그럼 일단 둬 급하니까 빨리 머지하자